src/base/static/components/place-detail/place-detail-survey-response.js

```diff
@@ -8,45 +8,59 @@ import constants from "../constants";
 
 import "./place-detail-survey-response.scss";
 
-const PlaceDetailSurveyResponse = props => {
-  return (
-    <article className="place-detail-survey-response">
-      <div className="place-detail-survey-response__body">
-        {props.surveyConfig.items
-          .filter(
-            field =>
-              field.type !== constants.SUBMIT_FIELD_TYPENAME &&
-              field.name !== constants.SUBMITTER_FIELDNAME
-          )
-          .map(field => (
-            <p
-              key={field.name}
-              className="place-detail-survey-response__paragraph"
-            >
-              {props.attributes.get(field.name)}
-            </p>
-          ))}
-      </div>
-      <section className="place-detail-survey-response__metadata-bar">
-        <Avatar
-          className="place-detail-survey-response__avatar"
-          src={props.submitter.avatar_url}
-        />
-        <section className="place-detail-survey-response__details-container">
-          <SubmitterName
-            className="place-detail-survey-response__submitter-name"
-            submitter={props.submitter}
-            placeConfig={props.placeConfig}
+class PlaceDetailSurveyResponse extends Component {
+  componentDidMount() {
+    if (this.props.modelId === parseInt(this.props.scrollToResponseId)) {
+      this.props.onMountTargetResponse(this.responseRef);
```

Comment:
Are you sure that `this.responseRef` is not null here?

Reply:
I confirmed that the ref is not null here. I think the issue might have to do with allowing queued rendering work to complete: see [https://github.com/mapseed/platform/pull/878/commits/94d7763b8a47d1d8832b48e06b8692f780611b05](this comment).